deckhouse-controller/pkg/controller/deckhouse-release/controller.go

```diff
@@ -381,7 +386,7 @@ func (r *deckhouseReleaseReconciler) pendingReleaseReconcile(ctx context.Context
 		return ctrl.Result{RequeueAfter: defaultCheckInterval}, nil
 	}
 
-	checker, err := d8updater.NewRequirementsChecker(r.client, r.moduleManager.GetEnabledModuleNames(), r.logger)
+	checker, err := releaseUpdater.NewDeckhouseReleaseRequirementsChecker(r.client, r.moduleManager.GetEnabledModuleNames(), r.logger)
```

Comment:
And why was it necessary to rename NewRequirementsChecker to NewDeckhouseReleaseRequirementsChecker?